src/default-settings.js

```diff
@@ -121,13 +132,30 @@ export const controlInputs =
     ];
 
 export function syncControlInputs(preControlInputs, preSettings) {
+    const value_colLabels =
```

Comment:
So basically, if these `value_cols` are included in `settings.details_cols` they get a custom `label`. Items in `settings.details_cols` that aren't listed below, just don't get a custom label. Yes?